lib/turn.rb

```diff
@@ -0,0 +1,20 @@
+class Turn
+  attr_reader :guess,
+              :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    @guess.capitalize == @card.answer.capitalize
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
This line can also be written as `if correct?` because the `if` is looking for a true or false result and `correct?` on its own is true (or false).